recipe/build.sh

```diff
@@ -150,7 +150,12 @@ if [[ $(uname) == "Linux" ]]; then
 #                -ltcg \
 #                --disable-new-dtags \
 
-  make -j${MAKE_JOBS}
+  if [ ${target_platform} == "linux-aarch64" ]; then
+    # Suppress logging.
+    make -j${MAKE_JOBS} > /dev/null
```

Comment:
seeing as azure takes much more than 1 hour. i doubt this will finish on travis anyway. While the logs are long, it is better than no logs at all....